doc/StereoStreaming.md

```diff
@@ -31,7 +42,7 @@ payload. This message is silently ignored by older Servers.
 
 ## Examples
 
-Example of a stereo 3D client application using the blocking Stream API:
+Example of a stereo 3D client application using synchronous operations:
```

Comment:
Actually you made me realize yesterday that std::future destructor does not wait(), execpt in the special case of std::async(std::launch::async). http://en.cppreference.com/w/cpp/thread/future/%7Efuture
So I think this is not the same as the previous synchronous api. In fact, if the sending takes longer than the rendering this will queue up send requests indefinitely, or more likely crash because an ImageWrapper will point to invalid data...